definitions/pipelines/immuno.cwl

```diff
@@ -276,6 +276,13 @@ inputs:
     pvacseq_threads:
         type: int?
 
+    immuno_tumor_sample_name:
+        type: string
+    immuno_tumor_sample_name_arr:
+        type: string[]
+    immuno_normal_sample_name:
+        type: string
+
```

Comment:
I'm confused as to why these can't be the same as the sample names, and just passed in as "tumor_sample_name" and "normal_sample_name".  Those are easy to provide via `process_inputs.pl` if using GMS, or in the yaml if running outside it

Reply:
There is also `tumor_name` and `normal_name` although I'm not sure how they are used because they are set to `tumor` and `normal` by default. I agree that we should consolidate things wherever possible.